server/models/buoy.py

```diff
@@ -1,5 +1,6 @@
 from geoalchemy2.types import Geography
 from sqlalchemy import Column, Float, Integer, SmallInteger, String
+from sqlalchemy.orm import relationship
```

Comment:
Locally in your code run `poetry run check`. You should get an error message regarding missing types for imports such as this one.

Please add `# type: ignore` [like so](https://github.com/clairBuoyant/server/blob/main/server/models/coastline.py#L5).

`poetry run check` should run without errors after updating those imports. Think of it like running spell check in Word.